Core/src/main/java/com/plotsquared/core/generator/HybridPlotWorld.java

```diff
@@ -456,8 +490,52 @@ public void addOverlayBiome(short x, short z, BiomeType id) {
         this.G_SCH_B.put(pair, id);
     }
 
+    /**
+     * Get the entities contained within the plot schematic for generation. Intended for internal use only.
+     *
+     * @since TODO
```

Comment:
Should be possible to hide these methods from generated javadoc with `@hidden`. Not sure if we want that though.

Reply:
People would still be able to access the method from the API though, so it'd be worth telling them what the method does still tbh

Reply:
-0 for hiding methods from javadocs, we should rather outline and document which methods can and should be used while outlining which stuff is internal or should rather not be used.